### ISSUE TITLE
Undo / Redo

### ISSUE THREAD

**Issue by Shabanovv** [mindbox-moscow/ui-kit#782]
Добавить кнопки назад/вперед. Ожидание: делают шаг назад на каждое удаление фильтра

**Comment by timramone** [comment]
Так, голосом обсуждали без меня, давайте для начала сделаем undo/redo для удаления, если заработает, добавить для других действий будет не сложно.

Идея в следующем: выделяем

1. Заменяем экшн удаления, который мы диспатчим из UI, на другой. Сейчас в `SimpleFilterCondition` и `GroupFilterCondition` используется `Actions.removeFilterCondition`, вместо этого добавляем новый `Actions.removeFilterConditionByUser`. Делаем сагу для этого экшна, она должна пока просто диспатчить `Actions.removeFilterCondition` и всё. Пишем тестик на это.
2. Делаем слайз стора для undo внутри filtersStore. В нём будет хранить

**Comment by Shabanovv** [comment]
На мне: макет с кнопками
Сливаем шорткаты (ctrl+z)

**Comment by Shabanovv** [comment]
Макеты https://invis.io/7FVLAPNAQ58#/401118522_4_-_Mouseover

![image](https://user-images.githubusercontent.com/41429186/78547425-c1380680-7807-11ea-8c62-d61cfb9dbf6a.png)

**Comment by igorFrontend** [comment]
Так же учитываем вставку кондишна, сдвиг всех дат, создание фильтра из сегмента

**Comment by Shabanovv** [comment]
когда удаляю весь фильтр целиком, потом не могу сделать шаг назад https://recordit.co/KtmzaUxVo1

**Comment by Shabanovv** [comment]
https://recordit.co/Ra4WHn505q
